app/src/main/java/lt/vilnius/tvarkau/fragments/SingleProblemMapFragment.kt

```diff
@@ -0,0 +1,72 @@
+package lt.vilnius.tvarkau.fragments
+
+import android.os.Bundle
+import android.os.Parcelable
+import com.google.android.gms.maps.GoogleMap
+import com.google.android.gms.maps.OnMapReadyCallback
+import com.google.android.gms.maps.model.Marker
+import lt.vilnius.tvarkau.entity.Problem
+import org.parceler.Parcels
+
+class SingleProblemMapFragment : BaseMapFragment(), OnMapReadyCallback, GoogleMap.OnInfoWindowClickListener, GoogleMap.OnInfoWindowCloseListener {
+
+    internal var problem: Problem? = null
+
+    override fun onActivityCreated(savedInstanceState: Bundle?) {
+        super.onActivityCreated(savedInstanceState)
+
+        getMapAsync(this)
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        if (arguments != null) {
```

Comment:
Arguments can't be null.
I think it could be moved to
```
val problem : Problem
  get() = Parcels.unwrap<Problem>(arguments.getParcelable<Parcelable>(null))
```